### PR TITLE
feat(lang): Support RStudio server

### LINKED ISSUE #490: feat(lang): Support RStudio server as the editor for R language
## Description

https://www.rstudio.com/products/rstudio/download-server/

RStudio also supports remote development with the help of RStudio server.

---

**Message from the maintainers**:

Love this enhancement proposal? Give it a 👍. We prioritise the proposals with the most 👍.

### PR DESCRIPTION
**Pull request by gaocegege**
Fix #490 

```py
def build():
    base(os="ubuntu20.04", language="r")
    install.r_packages([
            "remotes",
            "rlang",
        ])
    config.rstudio_server()
    shell("zsh")
```

```
NAME    ENDPOINT                        SSH TARGET      IMAGE           GPU     CUDA    CUDNN   STATUS          CONTAINER ID 
r-basic rstudio: http://localhost:34621 r-basic.envd    r-basic:dev     false   <none>  <none>  Up 12 minutes   1eb7d40e5a8a
```

Signed-off-by: Ce Gao <cegao@tensorchord.ai>

### CODE REVIEW

**Review by terrytangyuan — APPROVED** [review]
Looks great!

File: base-images/build.sh [hunk 1]
```diff
@@ -31,13 +31,15 @@ docker buildx build \
...
+# TODO(gaocegege): Support linux/arm64
 docker buildx build \
     --build-arg ENVD_VERSION=${ENVD_VERSION} \
     --build-arg ENVD_SSH_IMAGE=ghcr.io/tensorchord/envd-ssh-from-scratch \
     --build-arg HTTP_PROXY=${HTTP_PROXY} \
     --build-arg HTTPS_PROXY=${HTTPS_PROXY} \
-    -t ${DOCKER_HUB_ORG}/r-base:4.2  \
-    --pull --push --platform linux/x86_64,linux/arm64 \
+    -t ${DOCKER_HUB_ORG}/r-base:4.2 \
+    --pull --push --platform linux/x86_64 \
```
Comment (terrytangyuan) [inline base-images/build.sh:42]: We should probably automate the release of images as part of the release process

Comment (gaocegege) [reply base-images/build.sh:42]: SGTM! I will open an issue to keep track.

**Review by VoVAllen — APPROVED** [review]
LGTM

File: base-images/build.sh (hunk 1 above)
Comment (gaocegege) [reply base-images/build.sh:42]: #545

**Pull request merged**